opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -471,25 +470,13 @@ def set_metric_labels(series, view, tag_values):
             "TagKeys and TagValues don't have same size."
         )  # pragma: NO COVER
 
-    for ii, tag_value in enumerate(tag_values):
-        if tag_value is not None:
-            metric_label = remove_non_alphanumeric(view.columns[ii])
-            series.metric.labels[metric_label] = tag_value
+    for key, value in zip(view.columns, tag_values):
+        if value is not None:
+            series.metric.labels[remove_non_alphanumeric(key)] = value
     series.metric.labels[OPENCENSUS_TASK] = get_task_value()
 
 
 def remove_non_alphanumeric(text):
     """ Remove characters not accepted in labels key
     """
     return str(re.sub('[^0-9a-zA-Z ]+', '', text)).replace(" ", "")
```

Comment:
One other thing I noticed @c24t, this removes `_` from labels but they are valid characters and I think are important visually. I'll open a separate PR for this.

Reply:
And the same exporter in go replaces non-alphanumerics with underscores: https://github.com/c24t/opencensus-go-exporter-stackdriver/blob/281244432d1ddf7f0566cfd0964d79c2b6cea7c2/stats.go#L414-L423. Definitely a problem for another PR.